### PR TITLE
Contact part

### PR DESCRIPTION
**Pull request by NurkaAmre**
### Contact-form
- You must use for form `<ul>`
- Must used **display:flex**
- Fields must have placeholder.
- Message field must accept 500character, name field 30character, email must accept only email.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -216,6 +216,57 @@ <h3 class="skill-heading">Skills</h3>
           </ul>
         </div>
       </section>
+      <footer>
+        <div class="container container-information">
+           <div class="top-footer">
+              <h2 class="footer-heading">Get Started</h2>
+              <p class="last-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+              <form>
```
Comment (ichala) [inline index.html]: - [ ] Great job @NurkaAmre the website looks awesome 👏 , the form need to submit the data to[ Formspree service](https://formspree.io/html) , so kindly fix the issue by adding action and method attributes 

> action value should be similar to `https://formspree.io/f/{form_id}`
> method value equals POST

File: index.html [hunk 2]
```diff
@@ -216,6 +216,57 @@ <h3 class="skill-heading">Skills</h3>
...
+                    <label class="label1">Your email</label>
+                    <input class="input2" type="email" id="mail" name="user-email" required>
+                  </li>
+                  <li class="input-item">
+                    <label>message</label>
+                    <textarea id="textarea1" placeholder="Enter your message here..." required maxlength="500"></textarea>
+                  </li>
+                  <li class="input-item">
+                    <button class="cta-btn-last">Start collaboration</button>
+                  </li>
```
Comment (ichala) [inline index.html:240]: - [ ] kindly add styles for Button interactions such as hover, active (when it's pressed), disabled, and enable (default state)  as shown in the Figma template.

image

File: index.html [hunk 3]
```diff
@@ -216,6 +216,57 @@ <h3 class="skill-heading">Skills</h3>
...
+                    <label class="label1">Your name</label>
+                    <input class="input2" type="text" id="name" name="user-name" required maxlength="30">
+                  </li>
+                  <li class="input-item">
+                    <label class="label1">Your email</label>
+                    <input class="input2" type="email" id="mail" name="user-email" required>
+                  </li>
+                  <li class="input-item">
+                    <label>message</label>
+                    <textarea id="textarea1" placeholder="Enter your message here..." required maxlength="500"></textarea>
```
Comment (ichala) [inline index.html]: - [ ] kindly change the font color when typing data inside form elements, currently they are barely visible try something darker to make it easily visible

Before          |  After
:-------------------------:|:-------------------------:
texb |  texa

File: index.html [hunk 4]
```diff
@@ -216,6 +216,57 @@ <h3 class="skill-heading">Skills</h3>
...
+                  </li>
+                  <li class="input-item">
+                    <button class="cta-btn-last">Start collaboration</button>
+                  </li>
+                </ul>
+              </form>
```
Comment (ichala) [inline index.html]: - [ ] kindly to to follow the Figma design when it comes to padding and margins for the form elements and the border styles as well 
image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @NurkaAmre , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

-  Good Design ✅ 
-  No linter errors ✅

 ## Required Changes ♻️

- [ ] **_Check the comments below_**

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi Team , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**